mlflow/protos/protos_for_test/test_message.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto2";
```

Comment:
Is it possible to move this file under tests?

Reply:
Good point! Done!